fs2/src/main/scala/tofu/fs2/package.scala

```diff
@@ -0,0 +1,13 @@
+package tofu
+
+import _root_.fs2.Stream
+import tofu.lift.Lift
+
+package object fs2 {
```

Comment:
If we now have `fs2` package, why not rename `fs2Instances` to `fs2.instances`? I never liked the name.

Reply:
@catostrophe, neither do I like `fs2Instances`, but such a renaming would break bincomp.
@Odomontois, what do you think?

Reply:
Can we add an alias with the `@deprecated` annotation?

Reply:
@catostrophe, agree. I guess we should move `fs2Syntax` to `fs2.syntax` then as well?

Reply:
Even more, maybe we should move this syntax to tofu.streams.syntax abstracting `Stream`?

Reply:
We may think about interop packagw like zio's

Reply:
@Odomontois, do you mean refactor `zioInterop` package in the same way?

Reply:
Yes